mobile-widgets/qmlmanager.cpp

```diff
@@ -537,11 +536,9 @@ void QMLManager::saveCloudCredentials()
 	    !verifyCredentials(QMLPrefs::instance()->cloudUserName(), QMLPrefs::instance()->cloudPassword(), QMLPrefs::instance()->cloudPin()))
 		return;
 
-	s.beginGroup("CloudStorage");
-	s.setValue("email", QMLPrefs::instance()->cloudUserName());
-	s.setValue("password", QMLPrefs::instance()->cloudPassword());
-	s.setValue("cloud_verification_status", QMLPrefs::instance()->credentialStatus());
-	s.sync();
+	qPrefCloudStorage::set_cloud_storage_email(QMLPrefs::instance()->cloudUserName());
+	qPrefCloudStorage::set_cloud_storage_password(QMLPrefs::instance()->cloudPassword());
+	qPrefCloudStorage::set_cloud_verification_status(QMLPrefs::instance()->credentialStatus());
```

Comment:
I see absolutely zero value in changes like this.
We have code that works. You split over two commits changes that make that code work differently.
And that I now need to spend time figuring out if it breaks something or has unintended side effects.
How is that good use of my time?

Reply:
I see where you are heading...might be better to abandon the whole thing. 

Actually the code that "works" does not respond correctly to default values, something you (thanks) introduced into qPrefs.

Reply:
that's a fair criticism of my comment.

I withdraw the statements that there's "absolutely zero value".